src/main/grammars/ElmLexer.flex

```diff
@@ -121,6 +121,18 @@ ThreeQuotes = \"\"\"
     }
 }
 
+<TYPE_PENDING> {
```

Comment:
Maybe there's a better way to do this in JFlex by looking at token history (or tracking it on our own)?